lib/protocol/elementIdAttribute.js

```diff
@@ -19,7 +19,13 @@ let elementIdAttribute = function (id, attributeName) {
         throw new ProtocolError('number or type of arguments don\'t agree with elementIdAttribute protocol command')
     }
 
-    return this.requestHandler.create(`/session/:sessionId/element/${id}/attribute/${attributeName}`)
+    var command = '/attribute/'
+
+    if (this.options.useW3C && this.desiredCapabilities.browserName === 'firefox' && attributeName === 'value') {
```

Comment:
The problem is, that only geckodriver for Firefox support now W3C protocol. Chrome, IE, Safari and Edge use old Selenium JSONWire protocol at this moment.
If I want to use direct drivers without selenium I want to setup this in config one time and do not change depending on the browser. Maybe other name for this option will resolve this misunderstanding